common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -3731,6 +3731,24 @@
     ],
     "sqlState" : "42K0I"
   },
+  "SQL_SCRIPTING_LABEL_ERROR" : {
```

Comment:
not sure if it's important or not, but maybe we can check with Serge if we want to use `SQL_SCRIPTING_*` for error classes, or maybe stick to the simple `SQL_*` or maybe something else.